figures/metrics.py

```diff
@@ -91,7 +91,7 @@ class LearnerCourseGrades(object):
     TODO: Make convenience method to instantiate from a GeneratedCertificate
     """
 
-    def __init__(self, user_id, course_id, **kwargs):
+    def __init__(self, user_id, course_id, **_kwargs):
```

Comment:
I'm learning here, why do we need the underscore here?

Reply:
@melvinsoft `**kwargs` is unused. Pylint has a bug and cannot supress the warning: `unused-argument` until Pylint v2. The solution is to use another convention for ignored variables which is to use `_` prefix to denote an unused variable.